templates/careers/index.html

```diff
@@ -18,82 +18,109 @@
   <img src='https://pubads.g.doubleclick.net/activity;xsp=5024608;ord=1?' width=1 height=1 border=0>
 </noscript>
 
-<section class="p-strip--image is-dark is-deep" style="background-image: url(https://assets.ubuntu.com/v1/aebc1742-01_suru2_dark_2K.png); background-position: 100% 55%; background-size: 180%; background-repeat: repeat-x;">
-  <div class="row">
-    <div class="col-8 col-start-large-4">
-      <h1 class="p-heading--2">Make a difference in open source</h1>
-      {% if vacancies %}
-        {% include "careers/_search.html" %}
-      {% endif %}
+<div class="is-paper">
```

Comment:
In Vanilla `is-paper` is going to be a class on `body` (for full page background), maybe worth doing it like that here already as well, so it will be a seamless upgrade later.

Reply:
You mean adding `is-paper` to the body template? That's going to break some pages...

Reply:
In Vanilla we do it like that for examples:

<img width="611" alt="image" src="https://github.com/canonical/canonical.com/assets/83575/c8807e3d-b903-41fe-bdfd-f9f7c7c72e0c">


And then you can `{% set is_paper = True %}` in specific page template.

Reply:
Seems to work, thanks. Done